src/pages/Loans/LoansOverview/components/LoansInsights/LoansInsights.tsx

```diff
@@ -0,0 +1,36 @@
+import { Card } from '@/component-library';
```

Comment:
Fine here for now, but can you give some thought to how this could be made into a generic component? Broadly speaking, anything that is clearly scoped to a particular feature should be in the feature's components directory, but when an element is very generic-looking (as this one is: label and a value) then it's better in the component library.

Reply:
I dont think I get your idea. Card is the component library.

Reply:
Comment was meant for the whole file :) the Loan Insights section (assuming that it's the section across the top) is very generic, so I'd prefer it in the component library. Doesnt' have to be this PR though.